src/RoadRegistry.Syndication.ProjectionHost/Mapping/AtomEntrySerializerMapping.cs

```diff
@@ -0,0 +1,42 @@
+namespace RoadRegistry.Syndication.ProjectionHost.Mapping
+{
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.Runtime.Serialization;
+    using Projections;
+
+    public class AtomEntrySerializerMapping
+    {
+        private readonly IReadOnlyDictionary<string, DataContractSerializer> _atomEntrySerializers;
+
+        public AtomEntrySerializerMapping()
+        {
+            _atomEntrySerializers = new ReadOnlyDictionary<string, DataContractSerializer>(
+                new Dictionary<string, DataContractSerializer>
+                {
+                    {
+                        "https://data.vlaanderen.be/ns/gemeente",
+                        new DataContractSerializer(typeof(SyndicationContent<Gemeente>))
+                    },
+                    {
+                        "https://data.vlaanderen.be/ns/straatnaam",
+                        new DataContractSerializer(typeof(SyndicationContent<StraatNaam>))
+                    }
+                });
+        }
+
+        public DataContractSerializer GetSerializerFor(string categoryName)
+        {
+            if (_atomEntrySerializers.ContainsKey(categoryName))
+                return _atomEntrySerializers[categoryName];
+
+            throw new KeyNotFoundException($"Serializer voor categorie met naam '{categoryName}' niet gevonden.");
```

Comment:
This repository uses English, please stick to it. Translation at the edges e.g. api, db, ... are the only places we can deviate from this.